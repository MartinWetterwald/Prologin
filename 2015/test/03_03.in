20
240 120 3 -45 45 32 12 1 45 3 21 45 42 35 21 20 22 24 25 26
