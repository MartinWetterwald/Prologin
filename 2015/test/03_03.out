240 -120 -117 -48 90 -13 -20 -11 44 -42 18 24 -3 -7 -14 -1 2 2 1 1
